src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Zamok Webview API</title>
+</head>
+
+<body class="bg-[#F8FBFD] p-10">
```

Comment:
Please center the page content vertically in the viewport.  Full width on tablets + some left/right padding on larger screens.  Add a fullscreen background image (lock.jpg from Zamok repo).   Add our Hexagon Logos to webpack (zamok-basic-0.png, zamok-basic-1.png, zamok-basic-2.png from Zamok repo).  Create a logo component, that has three states: default, changed, submitted.  Use `zamok-basic-0.png` for default.  `zamok-basic-1.png` for changed.  and `zamok-basic-2.png` for submitted state.  On page load, show the default logo.  When the user enters text into the dispatch box, show the changed logo.  When they dispatch the action, show the submitted logo.  Please pace the logo in the top right corner, on the same line as the Zamok API Demo text, and the same font-size.